### ISSUE TITLE
Precheck script complains if a USB2 device is not present, even if a USB3 device is present

### ISSUE THREAD

**Issue by rodwsmith** [canonical/checkbox#1801]
### Bug Description

Changes to the server certification rules quite some time ago permit accepting submissions that have USB3 devices even if USB2 devices are not present. (Previously, both devices had to be present, except on systems with no USB3 support.) The canonical-certification-precheck script still enforces the old rules and should be updated.

### Cert-blocker Test Case

- [ ] cert-blocker

### To Reproduce

1. Run canonical-certification-precheck on a system with a USB3 drive plugged in but no USB2 device.

### Expected Result

The output should not complain about the missing USB2 device.

### Actual Result

The summary complains that the USB testing has failed.

### Environment

Any system that supports both USB2 and USB3 devices

### Relevant log output

```shell

```

**Comment by rodwsmith** [comment]
PR submitted, accepted, and merged; issue is now closed.

**Comment by rodwsmith** [comment]
Close issue
